src/platform/forms-system/src/js/components/AddressViewField.jsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/jsx-key */
```

Comment:
Is this `eslint-disable` needed?

Reply:
LOL I don't think I noticed that before. Yeah, I'll remove it